backend/LeetWars.Core/LeetWars.Core.BLL/Services/UserService.cs

```diff
@@ -134,12 +133,9 @@ public async Task<BriefUserInfoDto> GetBriefUserInfoByIdAsync(long id)
     {
         var user = await GetUserByExpressionAsync(user => user.Id == id);
 
-        if (user is null)
-        {
-            throw new ArgumentNullException("Not Found", new Exception("User was not found"));
-        }
-
-        return _mapper.Map<User, BriefUserInfoDto>(user);
+        return user is null
+            ? throw new ArgumentNullException("Not Found", new Exception("User was not found"))
+            : _mapper.Map<User, BriefUserInfoDto>(user);

```

Comment:
you can use ?? instead of ternary operator

Reply:
I don’t understand how we can apply it in this case
because we need to throw an exception

Reply:
var user = await GetUserByExpressionAsync(user => user.Id == id) ?? throw new ArgumentNullException("Not Found", new Exception("User was not found"))

_mapper.Map<User, BriefUserInfoDto>(user);